src/polyfill/index.js

```diff
@@ -0,0 +1,55 @@
+import util from "util";
+
+/* eslint-disable  max-params */
+export const getAssertionFn = (client) => (passed, actual, expected, message) => {
+  client.api.assert.ok(passed, message);
+};
+
+export default function(client) {
+  if (!client) {
+    return;
+  }
+  if (typeof client.assertion !== "function") {
+    client.assertion = getAssertionFn(client);
+  }
+
+  if (!client.desiredCapabilities) {
+    client.desiredCapabilities = client.settings.desiredCapabilities;
+  }
+}
+
+/* eslint-disable  no-invalid-this,prefer-spread */
+export const exportLegacyAssertions = function(LegacyAssertion, module) {
+  const assertion = function() {
+    const args = arguments;
+    const instance = new LegacyAssertion(this.client);
+
+    if (this.message === undefined) {
```

Comment:
why not just !this.message and !this.expected ?

Reply:
Because `message = null` and `expected = false` are valid values.